tests/xharness/Utilities/ProjectFileExtensions.cs

```diff
@@ -538,14 +539,15 @@ public static void FixInfoPListInclude (this XmlDocument csproj, string suffix,
 			var import = GetInfoPListNode (csproj, false);
 			if (import != null) {
 				var value = import.Attributes ["Include"].Value;
-				var fname = System.IO.Path.GetFileName (value.Replace ('\\', '/'));
+				var unixValue = value.Replace ('\\', '/');
```

Comment:
>'/' [](start = 41, length = 3)

This could also be `Environment.DirectorySeparator` or what is the exact name

Reply:
hmm I can give it a try, but if it is ran on Windows, it won't be a unix path, right?

Reply:
Yeah, I guess I am just mentioning it because unit tests then fail for me on Windows but that is not a big issue